db/migrate/20170306230913_create_person_roles.rb

```diff
@@ -0,0 +1,8 @@
+class CreatePersonRoles < ActiveRecord::Migration[5.0]
+  def change
+    create_table :person_roles do |t|
+      t.belongs_to :person, index: true
```

Comment:
I think I've added them in here. After searching I found an article describing how to add foreign keys on a DB level and this it what it mentioned. Kind of strange though because the schema file didn't change between commits so I feel like I could still be missing the mark.